Sources/TuistCore/Graph/ModelExtensions/Target+Core.swift

```diff
@@ -142,4 +142,8 @@ extension Target {
 
         return Array(sourceFiles.values)
     }
+
+    public var bundlesResources: Bool {
+        supportsResources || product == .staticFramework
```

Comment:
Let's create a PR for `XcodeProj` to update `supportResources` variable there.

Reply:
I don't understand how we can update `supportResources` without introducing breaking changes.
`XcodeProj` is a public repository, so it can be used in many external projects, right?
`Tuist` uses `XcodeProj 8.24.4` at the moment. The last version of `XcodeProj` is `8.24.11`

Reply:
> I don't understand how we can update supportResources without introducing breaking changes.
XcodeProj is a public repository, so it can be used in many external projects, right?

The primary consumer is Tuist. `XcodeProj` shouldn't add breaking changes, but change of behavior is not a breaking change. Let's create the `XcodeProj` PR.

Reply:
I was wrong. `supportResources` is declared in the [XcodeGraph](https://github.com/tuist/XcodeGraph) repository. tuist/XcodeGraph#84